modules/core/eks/README.md

```diff
@@ -45,137 +45,49 @@ Security:
 
 - Kyverno policies (policy as enforcement in k8s)
 
-
-## Inputs/Outputs
+## Explaining the attributes
 
 ### Input Paramenters
 
 #### Required
 
```

Comment:
would adding the link help?  https://seed-farmer.readthedocs.io/en/latest/manifests.html#a-word-about-datafiles

Reply:
Good idea